package.json

```diff
@@ -50,9 +50,9 @@
     "rimraf": "^2.5.2",
     "source-map-loader": "^0.1.5",
     "ts-loader": "^0.8.2",
-    "typescript": "^1.8.9",
+    "typescript": "^2.0.0",
     "typings": "~1.0.5",
```

Comment:
I switched to @types:
```
"@types/body-parser": "0.0.29",
    "@types/compression": "0.0.29",
    "@types/cookie-parser": "^1.3.29",
    "@types/es6-shim": "0.0.28",
    "@types/express": "^4.0.29",
    "@types/express-serve-static-core": "^4.0.29",
    "@types/mime": "0.0.28",
    "@types/node": "^4.0.30",
    "@types/serve-static": "^1.7.27",
````
I updated the `tsconfig` accordingly:

```
"include": [
    "node_modules/@types/**/*.d.ts",
    "src/**/*.ts"
  ],
  "exclude": [
    "node_modules"
  ],
```
But I'm getting so many errors related to types: `error TS2304: Cannot find name 'Map'.`

I'll look at this one later.